src/request/xhr.ts

```diff
@@ -60,7 +66,7 @@ export default function xhr<T>(url: string, options: XhrRequestOptions = {}): Re
 
 		let timeoutHandle: Handle;
 		request.onreadystatechange = function (): void {
-			if (request.readyState === 4) {
+			if (!isAborted && request.readyState === 4) {
```

Comment:
Shouldn't we handle (as in reject) `isAborted && request.readyState === 4)`?  Because obviously that is happening at runtime and that should result in a rejected promise, right?  With something like `resolve(new Error('An aborted request has been resolved'));`